executor/builder.go

```diff
@@ -1197,6 +1198,25 @@ func (b *executorBuilder) buildExplain(v *plannercore.Explain) Executor {
 		if b.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl == nil {
 			b.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl = execdetails.NewRuntimeStatsColl(nil)
 		}
+		// If the resource group name is not empty or "default", we could collect and display the RU
```

Comment:
why not display for default group>

Reply:
Because it won't calculate the RU due to https://github.com/tikv/client-go/blob/9d950905d743258e72894c9fb09dd6e453e0b9af/internal/client/client_interceptor.go#L96.

Reply:
Then why don't we calculate the RU for the default group?

Reply:
It's an expected behavior decided before. According to the [documentation](https://docs.pingcap.com/tidb/dev/tidb-resource-control#how-to-use-resource-control):

> If a user is not bound to a resource group or is bound to a default resource group, the user's requests are not subject to the flow control restrictions of TiDB. The default resource group is currently not visible to the user and cannot be created or modified. You cannot view it with SHOW CREATE RESOURCE GROUP or SELECT * FROM information_schema.resource_groups. But you can view it through the mysql.user table.

Reply:
we will make default group visible，ref https://github.com/pingcap/tidb/pull/41606